charts/cryosparc/values.yaml

```
@@ -0,0 +1,161 @@
+global:
+  imagePullSecret:
+    name: osc-registry
+    registry: docker-registry.osc.edu
+    username: robot$webservices-read
+    password:
+  nodeSelectorRole: webservices
+  storageClass: webservices-nfs-client
+slurmCluster: "ascend"
+license: ''
+project: ''
+workingDir: /opt/cryosparc_master
+env: []
+defaultCommand: {}
+containers: {}
+image:
+  repository: docker-registry.osc.edu/sciapps/cryosparc
+  tag:
+  pullPolicy: IfNotPresent
+
+mounts:
+  home: /users/{{ .Values.project }}/{{ include "osc.common.serviceAccountValue" . }}
+  rwDir:
+    project: /fs/ess/{{ .Values.project }}
```

I'd recommend get rid of `rwDir` and just call it `project`.
```suggestion
  project: /fs/ess/{{ .Values.project }}
```